.gitmodules

```diff
@@ -18,7 +9,11 @@
 	url = git://github.com/spring/pyunitsync.git
 [submodule "tools/pr-downloader"]
 	path = tools/pr-downloader
-	url = git://github.com/spring/pr-downloader.git
+	url = https://github.com/beyond-all-reason/pr-downloader.git
```

Comment:
There were some removals going on spring/pr-d. I wasn't sure why and replaced it just in case.
We haven't developed anything for pr-d. I'll replace it back.